starter_code/views/partials/albumCard.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="card" style="width: 33.333%;">
```

Comment:
maybe add the width property to the class card in your css file. This way you do not need to rewrite the width every time inside the hbs file.